tests/test.js

```diff
@@ -1067,3 +1067,21 @@ test.cb.failing('throws original lookup error if not recognized', t => {
 		t.end();
 	});
 });
+
+test('cache and query stats', async t => {
+	const cacheable = new CacheableLookup();
```

Comment:
Can you use the mocked resolver like in the tests above?

Reply:
I'm trying to understand how to use ```resolver```. 
Would it be enough if I just add it to the existing test like that?
```javascript
const cacheable = new CacheableLookup({resolver});
```

Reply:
Here's an example: https://github.com/GKosheev/cacheable-lookup/blob/ca90ec3857e219af14c7ec7c2b9580c5de5f2601/tests/test.js#L408-L430